invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/ItemDetails/components/ItemMetadata/ItemMetadata.js

```diff
@@ -1,16 +1,30 @@
 import React from 'react';
 import { Component } from 'react';
+import { generatePath } from 'react-router';
 import PropTypes from 'prop-types';
 import { Grid, Segment, Container, Header, Table } from 'semantic-ui-react';
 import { CreateNewLoanModal } from './components/CreateNewLoanModal';
-import { openRecordEditor } from '../../../../../common/urls';
-import { item as itemApi } from '../../../../../common/api';
+import { DeleteRecordModal } from '../../../components/DeleteRecordModal';
+import { BackOfficeURLS, openRecordEditor } from '../../../../../common/urls';
+import { loan as loanApi, item as itemApi } from '../../../../../common/api';
 import { invenioConfig } from '../../../../../common/config';
 import { EditButton } from '../../../components/buttons';
 
 import './ItemMetadata.scss';
 
 export default class ItemMetadata extends Component {
+  constructor(props) {
+    super(props);
+    this.deleteItem = props.deleteItem;
+  }
+
+  _handleOnRefClick(loanPid) {
+    const navUrl = generatePath(BackOfficeURLS.loanDetails, {
+      loanPid: loanPid,
+    });
+    window.open(navUrl, `_target_${loanPid}`);
```

Comment:
quick question, curiosity: why `_target_<loadnPid>` and not maybe `_blank`?

Reply:
If you click more than one link they will open in different browser tabs instead of the same.